Source/gmenu.cpp

```diff
@@ -73,45 +73,41 @@ bool gmenu_is_active()
 	return sgpCurrentMenu != nullptr;
 }
 
-static void gmenu_up_down(bool isDown)
+static void GmenuUpDown(bool isDown)
 {
-	int i;
-
 	if (sgpCurrItem == nullptr) {
 		return;
 	}
 	mouseNavigation = false;
-	i = sgCurrentMenuIdx;
-	if (sgCurrentMenuIdx) {
-		while (i) {
+	int i = sgCurrentMenuIdx;
+	if (sgCurrentMenuIdx != 0) {
+		while (i != 0) {
 			i--;
 			if (isDown) {
 				sgpCurrItem++;
-				if (!sgpCurrItem->fnMenu)
+				if (sgpCurrItem->fnMenu == nullptr)
 					sgpCurrItem = &sgpCurrentMenu[0];
 			} else {
 				if (sgpCurrItem == sgpCurrentMenu)
 					sgpCurrItem = &sgpCurrentMenu[sgCurrentMenuIdx];
 				sgpCurrItem--;
 			}
 			if ((sgpCurrItem->dwFlags & GMENU_ENABLED) != 0) {
-				if (i)
+				if (i != 0)
 					PlaySFX(IS_TITLEMOV);
 				return;
 			}
 		}
 	}
 }
 
-static void gmenu_left_right(bool isRight)
+static void GmenuLeftRight(bool isRight)
 {
-	int step, steps;
-
 	if ((sgpCurrItem->dwFlags & GMENU_SLIDER) == 0)
 		return;
 
-	step = sgpCurrItem->dwFlags & 0xFFF;
-	steps = (int)(sgpCurrItem->dwFlags & 0xFFF000) >> 12;
+	int step = sgpCurrItem->dwFlags & 0xFFF;
+	int steps = (int)(sgpCurrItem->dwFlags & 0xFFF000) >> 12;

```

Comment:
The (int) is of course of the devil and will cause nasal demons eventually. Don't know why it's even there.

There are multiple (int)s in this review.

Reply:
Since I didn't add them they are out of scope for this review.